ansible/maintenance.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Shutdown all vms on hypervisors and reset performance vms
+  hosts: performance-hypervisors
+  roles:
+    - hypervisor
+  vars:
+    ansible_user: root
+  tasks:
+    - name: sleep
+      pause:
+        minutes: 4
+
+- name: run liquibase steps
+  hosts: performance-candlepin-vm
+  environment:
+    JAVA_HOME: /usr/lib/jvm/java-1.8.0/
+  roles:
+    - maintenance
+  vars:
+    ansible_user: jenkins
+
+- hosts: performance-hypervisors
+  tasks:
+  - name: shutdown performance vms
+    virt: name={{perf_vm_domain}} command=shutdown
```

Comment:
If you look at ansible docs, they recently changed to the form @Lorquas mentions above. For everything, no matter how long or short the one-liner would be.